src/EnergyPlus/DataGlobals.hh

```diff
@@ -166,6 +166,8 @@ struct DataGlobal : BaseGlobalStruct
     int FDnumIterYears = 0;
     bool printConsoleOutput = true;
 
+    bool useCache = true;
```

Comment:
New ```DataGlobals``` flag to tell when we should be using the cache. Currently, only used to turn off the cache during unit tests.